app/scraper/sources/s3.py

```diff
@@ -53,3 +56,23 @@ def upload_image(self, f, filename):
             }
         )
         return self.get_file_url(filename)
+
+    def deleted_unused_imgs(self, people):
+        """
+        :param people: list of people records of everyone scraped from face_book
+        """
+        filename_offset = len(S3_LOCATION)
+        scraped_image_filenames = { (person['img'][filename_offset:] if 'img' in person else None) for person in people }
```

Comment:
Rather than adding `None` to the set, you can add a check after `for person in people` that will evaluate this beforehand: { person['img'][filename_offset:] for person in people if 'img' in person }